docker/dev.yml

```diff
@@ -99,6 +98,29 @@ services:
       - ./html/data:/import
     ports:
       - "127.0.0.1:${MONGO_EXPOSE_PORT:-27017}:27017"
+  keycloak:
+    # FIXME: Add keycloak to docker-compose.yml and/or docker/prod.yml
+    image: ghcr.io/openfoodfacts/scrypt-password-hash-provider:main@sha256:f8bccaca986d99efeebb9625b929af099b468399032ec4ee808fe14e4a5d079c
+    environment:
+      - KEYCLOAK_ADMIN
+      - KEYCLOAK_ADMIN_PASSWORD
+      - KEYCLOAK_BASE_URL
+      - KEYCLOAK_REALM_NAME
+      - PRODUCT_OPENER_DOMAIN
+      - PRODUCT_OPENER_OIDC_CLIENT_ID
+      - PRODUCT_OPENER_OIDC_CLIENT_SECRET
+    volumes:
+      - keycloak_data:/opt/keycloak/data
+      - ./conf/keycloak:/opt/keycloak/data/import
+    command: start-dev --import-realm --features="account3,declarative-user-profile"
```

Comment:
Could we import the realm in the base image so that we keep the keycloak functionality more self-contained?

Reply:
I think the realm needs to be different for local dev, staging, and production setup. This one's useful for local development, hence why I map it in the dev branch, but we shouldn't import one with static secrets for security sensitive environments. The base setup for staging/production might have a place in https://github.com/openfoodfacts/openfoodfacts-infrastructure/tree/ddf4070c9ae9188af4e263c12cf1aab5dfe7b18b/confs, though?